go-server-server/go/default.go

```diff
@@ -1162,6 +1162,133 @@ func ConfigVrouterVrfIdRoutesPatch(w http.ResponseWriter, r *http.Request) {
     }
 }
 
+func ConfigVrfVrfIdRoutesPatch(w http.ResponseWriter, r *http.Request) {
+    w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+    db := &conf_db_ops
+    vars := mux.Vars(r)
+    var rt_tb_key string
+    vrf_id_str := vars["vrf_id"]
+
+    if vrf_id_str != "default" {
```

Comment:
Why only default is supported. Is it a use case requirement or just we are not including others for this stage?

Reply:
`non-default` vrf requires vrf config APIs and validations. Since we don't have that immediate use-case, prefer to have it done for next phase with more thorough test cases